tests/Foundation/FoundationHelpersTest.php

```diff
@@ -78,4 +78,10 @@ public function testMixDoesNotIncludeHost()
 
         unlink(public_path('mix-manifest.json'));
     }
+
+    public function testFakerIsCreated()
+    {
+        $this->assertTrue(faker() instanceof \Faker\Generator);
+        $this->assertTrue(faker('fr_FR') instanceof \Faker\Generator);
```

Comment:
you can use `$this->assertInstanceOf('\Faker\Generator', faker());` here for clearer errors 🙂

Reply:
Oh nice, I’m camping for the weekend, but if there is interest in this and it hasn’t been canned by the boss, I’ll tweak this on Monday (Aus Time).
Thanks for the tip!